### ISSUE TITLE
Rendering of the wagtail.core.blocks.StaticBlock

### ISSUE THREAD

**Issue by leewesleyv** [noripyt/wagtail-react-streamfield#16]
Hi,

I've noticed an issue with the rendering of a StaticBlock using ```wagtail-react-streamfield==0.8.5``` in combination with ```wagtail==2.2.2```. 

In these 2 cases I would be presented with an input field instead:

```python
class LatestCoursesBlock(blocks.StructBlock):
    static = blocks.StaticBlock(admin_text="Latest courses: no configuration needed.")

    class Meta:
        template = 'latest_courses.html'
        icon = 'fa-graduation-cap'
```

<img width="1040" alt="screen shot 2018-10-15 at 17 47 44" src="https://user-images.githubusercontent.com/32985132/46962173-c862d600-d0a2-11e8-9dbd-fc3dbb37d6e4.png">

```python
class LatestCoursesBlock(blocks.StaticBlock):

    class Meta:
        template = 'latest_courses.html'
        admin_text = "Latest courses: no configuration needed."
        icon = 'fa-graduation-cap'
```

<img width="1039" alt="screen shot 2018-10-15 at 17 45 02" src="https://user-images.githubusercontent.com/32985132/46962196-d3b60180-d0a2-11e8-9b34-030634e615c6.png">

If you need more information I would be happy to supply it for you.

Thanks in regards!

**Comment by nuljon** [comment]
I have nearly identical experience with static block, instead of admin text I see empty input field in admin with wagtail v2.2.2:

class AuthorBlock(blocks.StaticBlock):

    class Meta:
        icon = 'user'
        label = 'Author Profile'
        admin_text = '{label} : is configured by User Profile snippet'.format(label=label)

**Comment by BertrandBordage** [comment]
Fixed by e364e712fd0bf38325fc534e4f4f0f244fadaa18 and released in 0.9.0.

**Comment by leewesleyv** [comment]
Awesome!
